packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -205,6 +206,20 @@ export default class AgoraIpOrderCtrl {
     this.trackStep(3);
   }
 
+  getServiceRegion() {
+    this.model.selectedServiceRegion = null;
+    if (
+      this.model?.selectedService?.type ===
+      PRODUCT_TYPES.dedicatedServer.typeName
+    ) {
+      this.IpAgoraOrder.getDedicatedServerRegion(
+        this.model.selectedService.serviceName,
+      ).then((region) => {
```

Comment:
if the call fails you should reset `selectedServiceRegion` to null in a `catch` block to avoid any side effect

Reply:
selectedServiceRegion is already reset at the beginning of the function. If an error is thrown, nothing will happen : has the value is null the div simply won't be displayed